aws/resource_aws_cloudwatch_event_rule_test.go

```diff
@@ -136,6 +136,36 @@ func TestAccAWSCloudWatchEventRule_prefix(t *testing.T) {
 	})
 }
 
+func TestAccAWSCloudWatchEventRule_tags(t *testing.T) {
+	var rule events.DescribeRuleOutput
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSCloudWatchEventRuleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSCloudWatchEventRuleConfig_tags,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCloudWatchEventRuleExists("aws_cloudwatch_event_rule.default", &rule),
+					resource.TestCheckResourceAttr("aws_cloudwatch_event_rule.default", "tags.%", "2"),
+					resource.TestCheckResourceAttr("aws_cloudwatch_event_rule.default", "tags.fizz", "buzz"),
+					resource.TestCheckResourceAttr("aws_cloudwatch_event_rule.default", "tags.foo", "bar"),
+				),
+			},
+			{
+				Config: testAccAWSCloudWatchEventRuleConfig_updateTags,
```

Comment:
Nit: it'd be great if the update configuration added or removed tag(s). Preferably having test steps that cover both scenarios.

Reply:
I added test case.